### PR TITLE
catch null openApi caused by parsing issues

### LINKED ISSUE #27: Wrong file encoding leads to unclear error message
**Setup information**
hikaku version: 1.1.1 (probably also in 2.0.0)
specification converter: OpenApi
OS: Windows 7 Enterprise

**Describe the bug**
If the open api specification file (e.g. `openapi.json`) is is read with other encoding than UTF-8 (happens on windows, apparently), then the following line in the OpenApiConverter results in an error, because `openApi` will be `null`:
```
 val openApi = OpenAPIV3Parser().readContents(specificationContent, null, null).openAPI
```
Though the encoding error is expected, it is shadowed by an `IllegalStateException` thrown by the illegal null access. The encoding error is not an exception, but a message inside the `SwaggerParseResult`

**Expected behavior**
The encoding  error should be presented to the user.

### PR DESCRIPTION
**Pull request by lmller**
closes #27

### CODE REVIEW

**Comment by lmller** [comment]
I don't think that there's a way to make the OpenApiParser use a different charset than utf-8

**Comment by cc-jhr** [comment]
File is read by hikaku. The content is passed as a String to the parser.
Can you re-request the review after making the changes we discussed today?

File: openapi/src/main/kotlin/de/codecentric/hikaku/converters/openapi/OpenApiConverter.kt [hunk 1]
```diff
@@ -24,8 +25,8 @@ import io.swagger.v3.oas.models.parameters.QueryParameter as OpenApiQueryParamet
  */
 class OpenApiConverter private constructor(private val specificationContent: String) : AbstractEndpointConverter() {
 
-    constructor(openApiSpecification: File): this(openApiSpecification.toPath())
-    constructor(openApiSpecification: Path): this(readFileContent(openApiSpecification))
+    constructor(openApiSpecification: File, charset: Charset = UTF_8): this(openApiSpecification.toPath(), charset)
+    constructor(openApiSpecification: Path, charset: Charset = UTF_8): this(readFileContent(openApiSpecification, charset))
```
Comment (cc-jhr) [inline openapi/src/main/kotlin/de/codecentric/hikaku/converters/openapi/OpenApiConverter.kt]: JvmOverload Annotation for Java users?

Comment (lmller) [reply openapi/src/main/kotlin/de/codecentric/hikaku/converters/openapi/OpenApiConverter.kt]: damn java

**Pull request merged**